.env.sample

```diff
@@ -9,12 +9,8 @@ CSP_SLACK_ACCESS_TOKEN=
 CSP_SLACK_STATUS_CHANNEL=
 CSP_SLACK_TRUNCATION=20
 
-CSP_CARD_NEUTRAL_COLOR=lightgray
-CSP_CARD_OK_COLOR=#abf5c3
 CSP_CARD_OK_EMOJI=white_check_mark
-CSP_CARD_WARN_COLOR=#f5e6ab
 CSP_CARD_WARN_EMOJI=warning
-CSP_CARD_ERROR_COLOR=#f4abaa
 CSP_CARD_ERROR_EMOJI=fire
```

Comment:
Why not make these configurable anymore?

Reply:
Oh, I see. This is some kind of Bootstrap Moment, innit?

Reply:
We could make them configurable, but they are pretty standard colors.  A combination of bootstrap classes and icons with colors baked in.